Wave/Wave/RowType.swift

```diff
@@ -13,4 +13,5 @@ public protocol RowType {
     
     func registerCells(tableView: UITableView)
     func configureCell(tableViewCell: UITableViewCell)
+    func didSelectCell(tableViewCell: UITableViewCell, tableView: UITableView, indexPath: NSIndexPath)
```

Comment:
Yeah I think so because after they select one I recall `configureCell` to reload where the checkmark is